hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -134,6 +141,23 @@ private void flushTransactions() {
     }
   }
 
+  /**
+   * Set OzoneManagerDoubleBuffer metrics values.
+   * @param flushedTransactionsSize
+   */
+  private void setOzoneManagerDoubleBufferMetrics(
+      long flushedTransactionsSize) {
+    ozoneManagerDoubleBufferMetrics.incTotalNumOfFlushIterations();
+    ozoneManagerDoubleBufferMetrics.setTotalNumOfFlushedTransactions(
```

Comment:
NIT: setTotalNumOfFlushedTransactions is confusing. Should we rename it to setTotalSizeOfFlushedTransactions

Reply:
Renamed the function as suggested.